docs/helm/index.md

```diff
@@ -0,0 +1,12 @@
+# Helm Charts
```

Comment:
Should we actually rename this to more generic `Deployment` (and the rename the subdir too) instead? Would probably be easier for people to navigate the documentation.

```suggestion
# Deployment
```

Then in this file shortly mention that currently only Helm charts are supported. We can add kustomize later if we're confident about that :)  WDYT?

Reply:
+1. Fixed.